driver-core/src/main/com/mongodb/internal/capi/MongoCryptHelper.java

```diff
@@ -85,18 +91,10 @@ public static MongoCryptOptions createMongoCryptOptions(final Map<String, Map<St
     public static MongoClientSettings createMongocryptdClientSettings(final String connectionString) {
 
         return MongoClientSettings.builder()
-                .applyToClusterSettings(new Block<ClusterSettings.Builder>() {
-                    @Override
-                    public void apply(final ClusterSettings.Builder builder) {
-                        builder.serverSelectionTimeout(1, TimeUnit.SECONDS);
-                    }
-                })
-                .applyToSocketSettings(new Block<SocketSettings.Builder>() {
-                    @Override
-                    public void apply(final SocketSettings.Builder builder) {
-                        builder.readTimeout(1, TimeUnit.SECONDS);
-                        builder.connectTimeout(1, TimeUnit.SECONDS);
-                    }
+                .applyToClusterSettings(builder -> builder.serverSelectionTimeout(1, TimeUnit.SECONDS))
```

Comment:
I don't mind the occasional Java 8 syntax update, but I'd prefer them kept to separate commits so it's clear which changes are functional and which are non-functional.  Would you mind reverting and doing this separately?

Reply:
Ack